playbooks/init/cluster_facts.yml

```diff
@@ -21,6 +21,7 @@
         https_proxy: "{{ openshift_https_proxy | default(None) }}"
         no_proxy: "{{ openshift_no_proxy | default(None) }}"
         generate_no_proxy_hosts: "{{ openshift_generate_no_proxy_hosts | default(True) }}"
+        dns_domain: "{{ dns_domain | default(None) }}"
```

Comment:
Lets rename this to `openshift_dns_domain` to be consistent with other options

Reply:
Done